dotcom-rendering/src/components/BylineLink.tsx

```diff
@@ -160,6 +161,10 @@ const getRenderedTokens = (
 		);
 	}
 
+	if (design === ArticleDesign.Crossword && renderedTokens.length > 0) {
+		return [<Fragment key="set-by">Set by: </Fragment>, ...renderedTokens];
```

Comment:
Its not clear why we need this Fragment? Also [keys](https://legacy.reactjs.org/docs/lists-and-keys.html#keys) are normally used to set a unique identifier for items in a list so not sure about the use case here.